packages/editor/src/components/zoom-out-toggle/index.js

```diff
@@ -27,11 +28,18 @@ const ZoomOutToggle = () => {
 		useDispatch( blockEditorStore )
 	);
 
+	const { setIsInserterOpened } = unlock( useDispatch( editorStore ) );
+
 	const handleZoomOut = () => {
 		if ( isZoomOut ) {
+			setIsInserterOpened( false );
 			resetZoomLevel();
 		} else {
 			setZoomLevel( 50 );
+			setIsInserterOpened( {
+				tab: 'patterns',
+				category: 'all',
```

Comment:
In my test, if I "toggle zoom-out" mode, the inserter opens with the "all" category selected. 

But If stay in zoom-out and close and then reopen the inserter, I'm not sure I understand why it shouldn't have the same effect (open "all" category).

The other question is where is the code responsible of enforcing "patterns" tab by default if I toggle the inserter in zoom-out? I think that code should do the same for "navigation" mode because otherwise, if we switch the "mode" when zoom-out is enabled, we kind of fallback to "blocks" by default.

Reply:
> But If stay in zoom-out and close and then reopen the inserter, I'm not sure I understand why it shouldn't have the same effect (open "all" category).

I think I subscribe to the intention behind the above, I am unsure if having the inserter button doing different things depending on the view is OK or confusing.

Reply:
> I think I subscribe to the intention behind the above, I am unsure if having the inserter button doing different things depending on the view is OK or confusing.

I'm also not sure 100% but I think maybe we should give it a shot separately and see.

What about my second question, do you have any idea?